docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,293 @@
+
+  * children: (list) If the block has child blocks, a list of IDs of the child
+    blocks. Returned only if ``children`` is included in the ``requested_fields``
+    query parameter.
+
+  * block_counts: (dict) For each block type specified in the ``block_counts``
+    query parameter, the aggregate number of blocks of that type within the
+    root block and all of its descendants. For example, if you specify
+    ``block_counts=video,problem`` as a query parameter, in the
+    ``block_counts`` response value the number of video blocks and problem
+    blocks in the specified block and in its children, is returned.
+
+  * graded: (boolean) Whether or not the block or any of its descendants is
+    graded. Returned only if ``graded`` is included in the ``requested_fields``
+    query parameter.
+
+  * format: (string) The assignment type of the block. Possible values can be
+    ``Homework``, ``Lab``, ``Midterm Exam``, and ``Final Exam``. Returned only if
+    ``format`` is included in the ``requested_fields`` query parameter.
+
+  * student_view_data: (dict) The JSON data for this block, if the specified
+    block implements the ``student_view_data`` method. The JSON data can be
+    used to natively render the XBlock. Returned only if the
+    ``student_view_data`` query parameter contains this block's type. See also
+    ``student_view_multi_device`` and ``student_view_url``.
+
+  * student_view_multi_device: (boolean) This value indicates whether or not
+    the HTML of the student view that is rendered at the ``student_view_url``
+    supports responsive web layouts, touch-based inputs, and interactive state
+    management for a variety of device sizes and types, including mobile and
+    touch devices. Returned only if ``student_view_multi_device`` is included
+    in the ``requested_fields`` query parameter.
+
+  * student_view_url: (string) The URL to retrieve the HTML rendering of the
+    block's student view. The HTML can include CSS and Javascript code. This
+    field can be used in combination with the ``student_view_multi_device``
+    field to determine whether a block can be viewed on a device. This URL can
+    be used as a fallback if the ``student_view_data`` for this block type is
+    not supported by the client or by the block.
+
+  * lms_web_url: (string) The URL to the navigational container of the XBlock
+    on the web LMS. This URL can be used as a fallback if the
+    ``student_view_data`` and ``student_view_url`` fields are not supported.
+
+  * lti_url: (string) The block URL for an LTI consumer. Returned only if the
+    ``ENABLE_LTI_PROVIDER`` Django setting is set to ``True``.
+
+
+============================================================================
+Example Response Showing a List of Course Blocks in a Specified Course
+============================================================================
+
+The following example response is returned from this request:
```

Comment:
May want to add
Note: since the depth parameter is not specified, only the top-level block in the course is returned.